### DISCUSSION TITLE
zsys and the Future of Ubuntu ZFS Support

### DISCUSSION THREAD

**Discussion by spflanze** [openzfs/zfs#13747] [Q&A]
In the ZFS install procedure at: 
https://openzfs.github.io/openzfs-docs/Getting%20Started/Ubuntu/Ubuntu%2022.04%20Root%20on%20ZFS.html
Under the heading "Overview / Ubuntu" I see:

`"The Ubuntu installer still has ZFS support, but it was almost removed for 22.04 and it no longer installs zsys. At the moment, this HOWTO still uses zsys, but that will be probably be removed in the near future."`

If Ubuntu drops ZFS support in a future version would that mean an Ubuntu installation that uses ZFS cannot be upgraded to it, and so an entirely new Ubuntu installation would have to be done?

Why was ZFS almost removed from 22.04? Would there be a successor in this case?

**Comment by gdevenyi** [comment]
This is a question for Ubuntu Launchpad, not OpenZFS.
